CHANGELOG.md

```diff
@@ -1,3 +1,52 @@
+<a name="4.1.0"></a>
+## [4.1.0](https://github.com/osquery/osquery/releases/tag/4.1.0)
+
+[Git Commits](https://github.com/osquery/osquery/compare/4.0.2...4.1.0)
+
+### New Features / Under the Hood improvements
+
+- Restore extension SDK and build support #5851
+- Documentation improvements #5860, #5852, #5912, #5954
+- Add more tests throughout the codebase #5837, #5832, #5857, #5864, #5855, #5869, #5871, #5885, #5903, #5879, #5914, #5941, #5957
+- Allow configuration more Linux Audit settings using flags #5953
+- Add logger_tls_max_lines flag #5956
+- Add AWS Session Token support #5944
+
+### Build
+
+- Lots of work on CPack-based packaging #5809, #5822, #5823, #5827, #5780, #5850, #5843, #5881, #5825, #5940, #5951, #5936
+- Lots of work porting Python2 to Python3 #5846
+- Use SQLite 3.29.0 on Windows and macOS #5810
+- Use aws-sdk-cpp source-builds on Windows and macOS #5889
+- Add various code quality checks and utilities #5834, #5730, #5872
```

Comment:
We could also add that we upgraded OpenSSL from 1.0.2o to version 1.0.2t (https://www.openssl.org/news/cl102.txt) and switched to the formula on all platforms

Reply:
Is it normal that there are no links generated for the hashtag shorthands in the preview under "Files changed"?

Reply:
That doesn't look correct :/